tests/adapters/cassettes/test_get_latest_videos_from_channel.yaml

```diff
@@ -0,0 +1,67 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept:
+      - application/json
+      Accept-Encoding:
+      - gzip, deflate
+      Connection:
+      - keep-alive
+      Content-Type:
+      - application/json
+      User-Agent:
+      - BusyBeaver
+    method: GET
+    uri: https://www.googleapis.com/youtube/v3/search?channelId=UCT372EAC1orBOSUd2fsA8WA&key=AIzaSyDmXRXSy-9JOmHnpH1HxojXUq1xrRoqunI&part=snippet%2Cid&order=date
```

Comment:
We can see the key in here; Not sure how to hide keys from VCR if the key is in a parameter. Could you look into this? I will also take a look over the next couple of days.

@philhenk heads up that since this is on GitHub, your youtube api key is compromised. no big deal as this is just for an api, but good to reset asap

Reply:
Thanks, I'll implement the change you recommend below and re-push.  

@alysivji thanks for pointing out the api key leak.  I received an alert from GitHub the following morning and reset it.